modulemd/tests/test-modulemd-modulestream.c

```diff
@@ -134,6 +134,77 @@ module_stream_test_arch (ModuleStreamFixture *fixture, gconstpointer user_data)
 }
 
 
+static void
+module_stream_v1_test_licenses (ModuleStreamFixture *fixture,
+                                gconstpointer user_data)
+{
+  g_autoptr (ModulemdModuleStreamV1) stream = NULL;
+  g_auto (GStrv) licenses = NULL;
+
+  stream = modulemd_module_stream_v1_new (NULL, NULL);
+
+  modulemd_module_stream_v1_add_content_license (stream, "GPLv2+");
+  licenses = modulemd_module_stream_v1_get_content_licenses_as_strv (stream);
+  g_assert_true (g_strv_contains ((const gchar *const *)licenses, "GPLv2+"));
+
+  g_clear_pointer (&licenses, g_strfreev);
+
+  modulemd_module_stream_v1_add_module_license (stream, "MIT");
+  licenses = modulemd_module_stream_v1_get_module_licenses_as_strv (stream);
+  g_assert_true (g_strv_contains ((const gchar *const *)licenses, "MIT"));
+
+  g_clear_pointer (&licenses, g_strfreev);
+
+  modulemd_module_stream_v1_remove_content_license (stream, "GPLv2+");
+  licenses = modulemd_module_stream_v1_get_content_licenses_as_strv (stream);
+  g_assert_cmpint (g_strv_length (licenses), ==, 0);
```

Comment:
This is wrong. After removal of `GPLv2+`, the length would be equal to 1 and not 0.

Reply:
It will be zero because [get_content_licenses_as_strv](https://fedora-modularity.github.io/libmodulemd/latest/modulemd-2.0-Modulemd.ModuleStreamV1.html#modulemd-module-stream-v1-get-content-licenses-as-strv) does not include "MIT" because "MIT" is a module license.

Reply:
Oh yes! My mistake. I'll just update the review comments then. Thanks for pointing that out.

Reply:
Updated.